pubspec.yaml

```diff
@@ -8,4 +8,4 @@ environment:
 dependencies:
```

Comment:
The SDK constraint ">=2.0.0-dev.22.0 <2.0.0" is about to break. Since we're here, let's bump that up to ">=2.0.0-dev.22.0 <3.0.0". (mit and I are investigating this issue for all packages)

You'll also want to increase the lower bound. I don't think implicit new is in dev.22.

Reply:
done